src/components/Imip.vue

```diff
@@ -157,6 +157,22 @@ function findAttendee(vEvent, email) {
 	return undefined
 }
 
+function findAttendeeByEmails(vEvent, emails) {
```

Comment:
Hey, we recently changed the logic with https://github.com/nextcloud/mail/pull/11915. Please ensure to rebase your branch and also re-test if that is still necessary.